exercises/minesweeper/description.md

```diff
@@ -1,25 +1,35 @@
-Add the numbers to a minesweeper board.
+Add the score totals to a completed Minesweeper board.
 
 Minesweeper is a popular game where the user has to find the mines using
 numeric hints that indicate how many mines are directly adjacent
 (horizontally, vertically, diagonally) to a square.
 
 In this exercise you have to create some code that counts the number of
-mines adjacent to a square and transforms boards like this (where `*`
-indicates a mine):
-
-    +-----+
-    | * * |
-    |  *  |
-    |  *  |
-    |     |
-    +-----+
-
-into this:
-
-    +-----+
-    |1*3*1|
-    |13*31|
-    | 2*2 |
-    | 111 |
-    +-----+
+mines adjacent to a given empty square and replaces that square with the
```

Comment:
will "a given" lead readers to think that a single square will be designated? Perhaps "each" instead?

Reply:
I'd say the example makes this self explanatory.